### ISSUE TITLE
Laravel 9 Support?

### ISSUE THREAD

**Issue by jomko** [BRACKETS-by-TRIAD/craftable#385]
Hi guys! Thank you for this awesome package. Do you plan to add Laravel 9 support?

**Comment by makin** [comment]
same problem here.

Problem 1
    - brackets/translatable v2.0 requires php ^7.2 -> your php version (8.0.15) does not satisfy that requirement.
    - brackets/translatable v2.1.0 requires php ^7.2.5 -> your php version (8.0.15) does not satisfy that requirement.
    - brackets/translatable 2.2.0 requires php ^7.2.5|^7.3.0 -> your php version (8.0.15) does not satisfy that requirement.
    - brackets/translatable[2.0.x-dev, ..., 2.3.0] require laravel/framework 6.*|7.*|8.* -> found laravel/framework[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but it conflicts with your root composer.json require (^9.0).
    - brackets/craftable[7.0.0, ..., 7.0.x-dev] require brackets/translatable ^2.0 -> satisfiable by brackets/translatable[v2.0, ..., 2.3.0].
    - Root composer.json requires brackets/craftable ^7.0 -> satisfiable by brackets/craftable[7.0.0, 7.0.x-dev].

**Comment by palypster** [comment]
We have released Laravel 9 support.

**Comment by annpinchuk** [comment]
```./vendor/bin/sail composer require --dev "brackets/craftable"
Info from https://repo.packagist.org: #StandWithUkraine
Using version ^8.0 for brackets/craftable
./composer.json has been updated
Running composer update brackets/craftable
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - phpoffice/phpspreadsheet[1.18.0, ..., 1.22.0] require psr/simple-cache ^1.0 -> found psr/simple-cache[1.0.0, 1.0.1] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - phpoffice/phpspreadsheet 1.23.0 requires psr/simple-cache ^1.0 || ^2.0 -> found psr/simple-cache[1.0.0, 1.0.1, 2.0.0, 2.x-dev] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - maatwebsite/excel[3.1.0, ..., 3.1.25] require php ^7.0 -> your php version (8.1.5) does not satisfy that requirement.
    - maatwebsite/excel[3.1.26, ..., 3.1.35] require illuminate/support 5.8.*|^6.0|^7.0|^8.0 -> found illuminate/support[v5.8.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - brackets/craftable[8.0.0, ..., 8.0.x-dev] require maatwebsite/excel ^3.1 -> satisfiable by maatwebsite/excel[3.1.0, ..., 3.1.x-dev].
    - maatwebsite/excel[3.1.36, ..., 3.1.x-dev] require phpoffice/phpspreadsheet ^1.18 -> satisfiable by phpoffice/phpspreadsheet[1.18.0, ..., 1.23.0].
    - Root composer.json requires brackets/craftable ^8.0 -> satisfiable by brackets/craftable[8.0.0, 8.0.x-dev].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require brackets/craftable:*" to figure out if any version is installable, or "composer require brackets/craftable:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```
